### PR TITLE
[bn] Localization of namespace.md

### PR DESCRIPTION
**Pull request by radiumSodium**

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@radiumSodium Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

**Comment by radiumSodium** [comment]
Thank you.
I have done the CLA part. If you have time, Would you mind check it agai.
Actually this is my first contribution, therefore I am a little afraid of
everything.

File: content/bn/docs/reference/glossary/namespace.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: নেমস্পেস(namespace)
+id: নেমস্পেস
```
Comment (asem-hamid) [inline content/bn/docs/reference/glossary/namespace.md:3]: id: namespace


you don't have to localize id

File: content/bn/docs/reference/glossary/namespace.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+date: ২০১৮-০৪-১২
```
Comment (asem-hamid) [inline content/bn/docs/reference/glossary/namespace.md:4]: 2028-04-12


you don't have to localize date

File: content/bn/docs/reference/glossary/namespace.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+full_link: /docs/concepts/overview/working-with-objects/namespaces
```
Comment (asem-hamid) [inline content/bn/docs/reference/glossary/namespace.md:5]: /bn/docs/concepts/overview/working-with-objects/namespaces

File: content/bn/docs/reference/glossary/namespace.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+short_description: >
+  একটি অ্যাবস্ট্রাকশন যা কুবার্নেটিস ব্যবহার করে রিসোর্স গ্রুপের আইসোলেশন সাপোর্ট করার জন্য, একটি একক ক্লাস্টার এর মধ্যে।
+
+aka: 
+tags:
+- fundamental
+---
+একটি অ্যাবস্ট্রাকশন যা কুবার্নেটিস ব্যবহার করে  রিসোর্স গ্রুপের আইসোলেশন সাপোর্ট করার জন্য, একটি একক {{< glossary_tooltip text="cluster" term_id="cluster" >}} এর মধ্যে।
```
Comment (asem-hamid) [inline content/bn/docs/reference/glossary/namespace.md:13]: {{< glossary_tooltip text="ক্লাস্টার" term_id="cluster" >}}

**Comment by asem-hamid** [comment]
maintain proper line breaks

image

**Review by mitul3737 — APPROVED** [review]

**Pull request merged**